### DISCUSSION TITLE
Images from Thursday discussion

### DISCUSSION THREAD

**Discussion by infinite0x20** [anhourandaquarter/carmina#5] [Ideas]
Hey y'all! Brainstorming happens here : )

**Comment by Emend026** [comment]
![IMG_0146](https://github.com/user-attachments/assets/41122c85-16f4-4d59-ae60-397592eb045c)

![IMG_0147](https://github.com/user-attachments/assets/c0d23cb1-a30b-4a06-beee-8b1dffb68f07)

**Reply by Emend026** [reply]
screenshots of our vision board
